cpp/thirdparty/versions.txt

```diff
@@ -25,35 +25,39 @@
 
 # Abseil LTS branch, Sept 2020, Patch 2
 ARROW_ABSL_BUILD_VERSION=0f3bb466b868b523cf1dc9b2aaaed65c77b28862
-ARROW_AWSSDK_BUILD_VERSION=1.8.90
+ARROW_AWSSDK_BUILD_VERSION=1.8.128
+# aws-checksums 0.1.10 has some issues it seems
 ARROW_AWS_CHECKSUMS_BUILD_VERSION=v0.1.5
 ARROW_AWS_C_COMMON_BUILD_VERSION=v0.4.59
 ARROW_AWS_C_EVENT_STREAM_BUILD_VERSION=v0.1.5
 ARROW_BOOST_BUILD_VERSION=1.71.0
-ARROW_BROTLI_BUILD_VERSION=v1.0.7
+ARROW_BROTLI_BUILD_VERSION=v1.0.9
 ARROW_BZIP2_BUILD_VERSION=1.0.8
-ARROW_CARES_BUILD_VERSION=1.16.1
+ARROW_CARES_BUILD_VERSION=1.17.1
 ARROW_GBENCHMARK_BUILD_VERSION=v1.5.2
 ARROW_GFLAGS_BUILD_VERSION=v2.2.2
 ARROW_GLOG_BUILD_VERSION=v0.4.0
 ARROW_GRPC_BUILD_VERSION=v1.33.2
 ARROW_GTEST_BUILD_VERSION=1.10.0
 ARROW_JEMALLOC_BUILD_VERSION=5.2.1
 ARROW_LZ4_BUILD_VERSION=v1.9.2
```

Comment:
Looks like this can be bumped to 1.9.3.

Reply:
Newer LZ4 also appears to fail on Visual Studio 2015: https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/builds/37520391/job/x4j7i0271o1rk7ih#L884

Reply:
Fixed.